test/unit/strategy.test.ts

```diff
@@ -163,8 +163,8 @@ describe('Deployed Strategy', () => {
   });
 
   it('can encrypt and decrypt', async () => {
```

Comment:
Fails with error:
```
decryption failed: Decryption of ciphertext failed: either someone tampered with the ciphertext or you are using an incorrect decryption key.
```
on line 215:
```
const decryptedMessage = await decrypter.retrieveAndDecrypt(
  [encryptedMessageKit],
  bobProvider
);
```
This test worked before the update. It doesn't use any premade test data (`testVariables.ts`). 
@theref Does that ring any bells?

Reply:
Looks like the state is leaking between the tests due to uncleared `jest.spy`.